calculator/pom.xml

```diff
@@ -85,6 +117,18 @@
             <groupId>io.grpc</groupId>
             <artifactId>grpc-protobuf</artifactId>
         </dependency>
+        <!-- Grpc-Test-Support -->
+        <dependency>
+            <groupId>com.asarkar.grpc</groupId>
+            <artifactId>grpc-test</artifactId>
+            <version>${asarkar}</version>
```

Comment:
no consistency in dependency definition - probably it would be a good idea to extract all versions in dependency management and take the version from there

Reply:
Asarcar grpc-test library is unique for the calculator service, do you mean to extract it anyway

Reply:
If this is the case probably it could stay here

Reply:
ok